src/epd3in7/command.rs

```diff
@@ -0,0 +1,89 @@
+//! SPI Commands for the Waveshare 3.7" E-Ink Display
+#![allow(non_camel_case_types)]
```

Comment:
Please remove the ignore non camel case types and switch the command names to CamelCase type similar as elsewhere in the repository (e.g. https://github.com/caemor/epd-waveshare/blob/main/src/epd2in13bc/command.rs)

Reply:
I should have read your comment first :see_no_evil: :-D

Reply:
Ohh, no worries, done :-)